app/views/papyrus/admin/templates/edit.html.slim

```diff
@@ -77,3 +77,5 @@
           .grid.grid-cols-12.gap-4
             .col-span-12
               = f.input :metadata, as: :editor, input_html: { value: YAML.dump(f.object.metadata) }, mode: 'application/yaml'
+        - t.tab :papers, badge: @template.papers.size
```

Comment:
We're phasing out `sts.table` for `card.table`.
Can you replace the tabs in template/edit with: `card.tab` so that these get directly created on the card and the call `card.table`, this will add a tab, similar to say:

```
= card.table(:order_lines_pricing_and_value, parameters: {order_id: order.id}, custom_views: false, tab: :pricing_and_value)
```

custom_views should be false and the tab: :papers

Reply:
updated